src/schema/v2/consignments/createConsignmentInquiryMutation.ts

```diff
@@ -29,6 +29,11 @@ const ConsignmentInquiryType = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLNonNull(GraphQLString),
       description: "Email of inquirer",
     },
+    recipientEmail: {
+      type: GraphQLString,
+      description:
+        "an optional Collector services team memeber email who was sent the Inquiry",
```

Comment:
You could use a tool like [Notion AI's "Fix spelling & grammar"](https://www.notion.so/product/ai) or a Raycast extension like [Correct Me](https://github.com/olerichter00/raycast-correct-me) to 

**Correct Me Suggestion:**
```suggestion
        "An optional email from a member of the Collector Services team to whom the request was sent",
```

**Notion AI Suggestion:**

```suggestion
        "An optional Collector Services team member email who was sent the inquiry."

Reply:
Thanks for the suggestion Ole, corrected